src/modules/ChannelList/components/ChannelPreview/utils.js

```diff
@@ -51,13 +50,18 @@ export const getTotalMembers = (channel) => (
     : 0
 );
 
-const getPrettyLastMessage = (message = {}) => {
+const getPrettyLastMessage = (message = null) => {
   const MAXLEN = 30;
-  const { messageType, name } = message;
-  if (messageType === 'file') {
-    return truncateString(name, MAXLEN);
+  if (!message) return '';
+  if (message.isFileMessage()) {
+    return truncateString(message.name, MAXLEN);
+  }
+  if (message.isMultipleFilesMessage()) {
+    const { fileInfoList } = message;
+    if (fileInfoList.length > 0) return fileInfoList[0].fileName;
```

Comment:
It's better to use the function
```suggestion
    if (fileInfoList.length > 0) {
      return truncateString(fileInfoList[0].fileName, MAXLEN);
    }
```

Reply:
It will reduce the string length if it's too long